packages/runner-sdk/lib/action.ts

```diff
@@ -355,7 +356,7 @@ export abstract class NangoActionBase {
         providerConfigKey: string,
         connectionId: string,
         sync: string | { name: string; variant: string },
```

Comment:
is there any reason why the runner-sdk `triggerSync` function doesn't have the same signature as the node-client one? here we can only trigger a single sync while with the node-client one can pass an array of sync.
@bodinsamuel @khaliqgant @bastienbeurier 

This PR is not addressing this problem but I am curious

Reply:
Not that I am aware of, I think I was barely here when it got introduced